scripts/policy/protocols/conn/known-services.zeek

```diff
@@ -39,6 +43,11 @@ export {
 	## operation.
 	const use_service_store = F &redef;
```

Comment:
Should we've deprecated this one with a comment that users should switch over to and test  `use_storage_framework` , would be an incentive to remove the code at 8.1.

Reply:
Both flags are necessary. If `use_service_store` is false, the services are stored in a table in memory instead of being backed up to a store. `use_storage_framework` switches between the two storage modes if `use_service_store` is true.